amt/site/templates/projects/new.html.j2

```diff
@@ -22,11 +23,11 @@
                                     class="utrecht-form-field utrecht-form-field--text rvo-form-field"
                                 >
                                     <div class="rvo-form-field__label">
-                                        <label class="rvo-label">{% trans %}Project name{% endtrans %}</label>
+                                        <label class="rvo-label">{% trans %}Project{% endtrans %}</label>
                                     </div>
                                     <input
                                         id="name"
-                                        placeholder="Jouw project naam hier"
+                                        placeholder="Jouw projectnaam hier"
```

Comment:
To be consistent this needs to be in English as well, since all our strings in the jinja temaples are English. Also can add translation.